src/main/webapp/resources/js/Controller/applicantController.js

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+angular.module('applicantApp')
+.controller('mainController',['$http','applicantUtilService','$scope',function($http,$scope,applicantUtilService,$location,$sce){
+	
+	
+	$scope.initEmp = function()
+	{
+		$scope.skillset=[];
+		$scope.empobj = {};
+		$scope.Allemps = [];
+		$scope.newprojects = [];
+		$scope.neweditprojects =[];
+		$scope.employee={};
+		$scope.empProjs = [];
+		$scope.empProj = {};
+	};
+	
+	$scope.skillset=[];
+	$scope.empobj = {};
+	$scope.Allemps = [];
+	$scope.newprojects = [];
+	$scope.neweditprojects =[];
+	$scope.employee={};
+	$scope.empProjs = [];
+	$scope.empProj = {};
+	
+	$scope.projectscount = 0;
+	$scope.baseUrl = '/resumeprocessing';
+	/*$scope.skillset=[];
+	$scope.empobj = {};
+	$scope.Allemps = [];
+	$scope.newprojects = [];
+	$scope.neweditprojects =[];
+	$scope.employee={};*/
+	$scope.searchObj = {};
+	$scope.showsearchdiv=false;
+	$scope.showempResume = false;
+	$scope.Viewall = false;
+	$scope.ViewSelected = false;
+	$scope.param ='';
+	$scope.searchCriteria='search By Name';
+	
+	$scope.techSkills={
+		"Languages":["C","C++","JAVA","Python","C#","Perl"],
+		"SouceControlTools":["Git","Clear Case","SVN"],
+		"WebTechnologies" :["html","css","Jquery","JSP","Angular JS","node.js","backbonejs","Ruby on Rails"],
+		"OperatingSystems":["Windows","Linux","Solaris","Unix","IBM","Redhat"]
+	};
```

Comment:
Better maintain separate Source to read this data. Or else make an API call to fetch this data from server itself.